template_overlay_typescript_components/src/app.tsx

```diff
@@ -25,7 +25,7 @@ export default class MyApp extends React.Component<Props, State> {
     return (
       <View name="main-view">
         <Text textSize={0.1} localPosition={[-0.3, 0, 0]}>
-          {this.state.message}
```

Comment:
If we don't use `this.state.message` then we no longer need the following lines:
```
state: State;
  constructor(props: Props) {
    super(props);
    this.state = {
      message: props.message
    };
  }
```